lib/jazzy/sourcekitten/Rakefile

```diff
@@ -5,10 +5,10 @@ task :prepare do
   Dir.glob('*.framework').each do |fw|
     p fw
     Dir.chdir(fw) do
-      basename = File.basename(fw, '*.framework')
-      version = Dir.glob('Versions/*').sort.last
```

Comment:
That totally works, but doesn’t it add a bit of indirection? We already have a number of things hardcoded in this Rakefile, so it will inevitably at some level be inflexible to changes in SourceKittens structure.